core/src/main/java/tc/oc/pgm/listeners/AntiGriefListener.java

```diff
@@ -90,6 +90,13 @@ private void participantDefuse(Player player, Entity entity) {
                     "moderation.defuse.player",
                     clicker.getBukkit(),
                     owner.getBukkit().getDisplayName(clicker.getBukkit()) + ChatColor.RED));
+        ChatDispatcher.broadcastAdminChatMessage(
+            TranslatableComponent.of(
+                "moderation.defuse.message",
+                TextColor.RED,
+                clicker.getName(NameStyle.FANCY),
+                owner.getName(NameStyle.FANCY)),
```

Comment:
`owner` is annotated as `@Nullable` so you need to account for that

Reply:
If the owner is found to be null, a seperate message including the defusing players name is being sent now